packages/ses/scripts/hermes.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+OS="$(uname -s)"
+
+case "$OS" in
+    Linux*)
+        OS_DIR="linux64-bin"
+        ;;
+    Darwin*)
+        OS_DIR="osx-bin"
+        ;;
+    CYGWIN*|MINGW*|MSYS*)
+        OS_DIR="win64-bin"
+        ;;
+    *)
+        echo "Unsupported OS: $OS"
+        exit 1
+        ;;
+esac
+
+# Paths relative to 'packages/ses'
+HERMESC="../../node_modules/hermes-engine-cli/$OS_DIR/hermesc"
+HERMES="../../node_modules/hermes-engine-cli/$OS_DIR/hermes"
+
+echo "Concatenating: dist/ses-hermes.cjs + test/hermes-smoke.js"
+cat dist/ses-hermes.cjs test/hermes-smoke.js > test/hermes-smoke-dist.js
```

Comment:
How is `dist/ses-hermes.cjs` intended to be used in practice? Is Hermes code expected to ever import it or is concatenation (or bundling with Rollup or is `@endo/bundle-source` an option? Can you take advantage of `-C` package conditions? If it’s only concatenation, we might opt to not to export `dist/ses-hermes.cjs` in `package.json` since it would never be retrieved through module resolution.

Reply:
the intended use is `require.resolve('ses/hermes')` in
- https://github.com/LavaMoat/LavaMoat/pull/1438
  - build.js (stuff to export)
  - makeGetPolyfills.js ([example](https://github.com/LavaMoat/LavaMoat/pull/1438/files#diff-67ace51464e7821ba8995de3da595f569da4b1b30b927422f4e97ff1b4d43cbdR10-R16))

which plugs into the Metro serializer at [getpolyfills](https://metrobundler.dev/docs/configuration/#getpolyfills)

so that we lockdown earlier w/o needing to [patch](https://github.com/MetaMask/metamask-mobile/blob/main/patches/react-native%2B0.72.15.patch#L1-L110) React Native core

Reply:
devil's advocate: for `@lavamoat/react-native-lockdown` i guess we could instead curl/wget the ses-hermes shim then require it w/o module resolution, but it becomes a pain to manually update it (vs exporting it for renovate/dependabot to pickup), and wouldn't be consistent resolving `ses` as a module but ses-hermes as a local file path

Reply:
however the concatenation is perfect here for our smoke test running in CI on Hermes VM, since Hermes doesn't support I/O